packages/client/src/components/Location/LocationInput.tsx

```diff
@@ -87,11 +81,7 @@ const LocationInput: React.FC<{
       ]);
       checkerContext.autofillData.address = address;
 
-      if (focus) {
-        (document.activeElement as HTMLElement).blur();
-      } else {
-        handleNewAddressSubmit();
-      }
+      handleNewAddressSubmit();
```

Comment:
We used to have the focus check, to disable the active keyboard on mobile. 

I will check if this still works on mobile.